backend/Api/Staffing/ReadModelFactory.cs

```diff
@@ -93,7 +93,7 @@ public List<ConsultantReadModel> GetConsultantReadModelsForWeeks(string orgUrlKe
             .Where(absence => weekSet.Contains(new Week(absence.Year, absence.WeekNumber)))
             .GroupBy(absence => absence.Absence.Name)
             .Select(grouping => new DetailedBooking(
-                new BookingDetails(grouping.Key, BookingType.PlannedAbsence),
+                new BookingDetails(grouping.Key, BookingType.PlannedAbsence, grouping.Key),
```

Comment:
Hvorfor brukes grouping.Key to ganger her? 😅

Reply:
Fordi absence ikke har forskjell på kundenavn og prosjektnavn. Alternativet kan være tom streng på prosjektnavn